code/Extension/UpgradePolymorphicExtension.php

```diff
@@ -22,8 +23,6 @@
  * instances, and were made polymorphic in SilverStripe 4 (which also requires a class name). This means that a
  * certain amount of manual checking is required to ensure that upgrades are performed smoothly.
  *
- * @internal This API is likely to be removed in later major versions of silverstripe/userforms
```

Comment:
"likely to be removed in later major versions" isn't what `@internal` is for, so I've removed that annotation as it might result in other changes being hidden from the tool that looks for API breaking changes.